do.sh

```diff
@@ -83,30 +83,22 @@ goget () {
 }
 
 coverage() {
-  # test all the things
-  if [ -n "$*" ]; then
-    go test -v -cover $* -coverprofile coverage.out
-  else
-    go test -v -cover ./... -coverprofile coverage.out
-  fi
+  export EXTRA_TEST_ARGS='-cover'
+  test
   go tool cover -html=coverage.out -o coverage.html
 }
 
 test () {
-  if [ -n LASSO_CONFIG ]; then
-    export LASSO_CONFIG=${GOPATH}/src/github.com/LassoProject/lasso/config/test_config.yml
+  if [ -z "$VOUCH_CONFIG" ]; then
+    export VOUCH_CONFIG="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/config/test_config.yml"
```

Comment:
the `cd dirname ${BASH_SOURCE}` is already accomplished at the top of ./do.sh
https://github.com/vouch/vouch-proxy/blob/master/do.sh#L4-L8

Reply:
Fixed.